docs/how-to-use-gvfs.md

```diff
@@ -223,3 +228,49 @@ In order for tensorflow to support GVFS, you need to recompile the [tensorflow-i
    ## list directory
    print(tf.io.gfile.listdir('gvfs://fileset/test_catalog/test_schema/test_fileset_1/'))
    ```
+
+## Authentication
+
+Currently, Gravitino Virtual File System supports two kinds of authentication types to access Gravitino server: `simple` and `oauth2`.  
+`simple` type is the default authentication type in Gravitino Virtual File System.
+
+### How to use
+#### Use `simple` authentication type
+
+Please make sure that your Gravitino server also configure the `simple` authentication mode firstly.  
```

Comment:
How to configure the user name in the simple mode?

Reply:
Add it in the code example.